lib/util.ml

```diff
@@ -1753,7 +1753,36 @@ let hexa_string s =
   done;
   Bytes.unsafe_to_string s'
 
-let print_alphab_list conf crit print_elem liste =
+let sort_by_date base pl =
```

Comment:
build error because sort_by_date defined here but not used, as line 1774 is a commented line.